File: tools/Issue558Detector/Issue558Detector/ServiceControlModel/EventLogItem.cs
using System;

namespace Issue558Detector
{
    public class EventLogItem
    {
        public string Id { get; set; }
        public string Description { get; set; }
        public int Severity { get; set; }
        public DateTime RaisedAt { get; set; }
        public string[] RelatedTo { get; set; }
        public string Category { get; set; }
        public string EventType { get; set; }
    }
}

File: frontend/src/resources/EventLogItem.ts
export default interface EventLogItem {
  id: string;
  description: string;
  severity: Severity;
  raised_at: string;
  related_to: string[];
  category: string;
  event_type: string;
}

export enum Severity {
  Critical = "critical",
  Error = "error",
  Warning = "warning",
  Info = "info",
}
